scipy/linalg/_basic.py

```diff
@@ -68,10 +68,13 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
         Square input data
     b : (N, NRHS) array_like
         Input data for the right hand side.
-    sym_pos : bool, optional
-        Assume `a` is symmetric and positive definite. This key is deprecated
-        and assume_a = 'pos' keyword is recommended instead. The functionality
-        is the same. It will be removed in the future.
+    sym_pos : bool, optional, deprecated
+        Assume `a` is symmetric and positive definite.
+
+        .. deprecated:: 0.19.0
+        This key is deprecated and ``assume_a = 'pos'`` keyword is recommended
+        instead. The functionality is the same. `sym_pos` will be removed in
+        SciPy 1.11.0.
```

Comment:
```suggestion
        .. deprecated:: 0.19.0
           This key is deprecated and ``assume_a = 'pos'`` keyword is recommended
           instead. The functionality is the same. `sym_pos` will be removed in
           SciPy 1.11.0.

```
To fix the CircleCI doc build warning:
```
/home/circleci/repo/build/testenv/lib/python3.8/site-packages/scipy/linalg/_basic.py:docstring of scipy.linalg._basic.solve:35: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```